### ISSUE TITLE
few sugestions

### ISSUE THREAD

**Issue by XeromX** [blissd/fotema#196]
I like your app it has huge potential but I have few suggestions:
1. possibility to add multiple folders with pictures and add filters to for example ignore small files or certain types of files like ico
2. preload next picture while browsing in "full screen" to make things faster
3. allow to change miniature size

**Comment by blissd** [comment]
Hi @XeromX. Thank you for your suggestions. However, I don't quite follow what the last one means. What is the miniature size?

**Comment by forteller** [comment]
I would guess 3 means to allow zooming in and out on the list of images, just like you can in Nautilus?

**Comment by 993782githbcom** [comment]
YES, I need this feature. Do you use Aves on Android?I think it's the best Android gallery, you can have a look: https://f-droid.org/packages/deckers.thibault.aves.libre/
